front/js/components/order.es6

```diff
@@ -42,7 +42,6 @@
       'onCartUpdate', renderTable, fillSavedInputs,
       touchSpinReinit, restoreSelectedPayment, cityAutocomplete,
     );
-    $(DOM.fullForm).submit(() => mediator.publish('onOrderSend'));
```

Comment:
The `submit` function from `submitOrder` provides a window  at 100ms for `onOrderSend` processing, but because of this line `onOrderSend` didn't use the window and executes after  

So `onOrderSend` must not subscribe on `DOM.fullForm` submit and executes before it, otherwise it won't use the window in 100ms